nexus/src/app/background/tasks/support_bundle_collector.rs

```diff
@@ -593,6 +611,12 @@ impl BundleCollection {
             }
         }
 
+        if let Err(e) = sp_dumps_fut.await {
```

Comment:
Most minor of nit; still seems weird this is "sandwiching" the sled-querying code. Why not immediately use `sp_dumps_fut` next to where it is declared?

Reply:
My initial thinking was to allow both the sled and SP tasks to run concurrently, but the SP calls should be very quick so I don't think we're really getting any benefit. Moved the await up top as you've suggested.